src/rsa/bigint.rs

```diff
@@ -218,6 +218,21 @@ pub struct Modulus<M> {
     ring: PhantomData<M>,
 }
 
+impl<M> Modulus<M> {
```

Comment:
> Why would we not allow this? JWK also has a private key mapping: https://tools.ietf.org/html/rfc7517#appendix-A.2

In *ring*, when you construct a private or symmetric key object, there's no way to get the key material out. This way, you can safely pass an `&RSAKeyPair` or `&OpeningKey` to a function and know that that function won't ever leak the key.

If you look at `Ed25519KeyPair::generate_serializable`, you can see that when we generate a key pair, we also output the bytes needed to serialize the key, along with the generated key pair. When we add RSA key generation to *ring*, we'll do it the same way.